lib/optimizely_server_side/helpers/support.rb

```diff
@@ -24,17 +24,43 @@ module Support
     def experiment(experiment_key, options = {}, &blk)
       # Merge any other options to user attributes
       OptimizelyServerSide.configuration.user_attributes.merge!(options.stringify_keys)
-      variation_key = optimizely_sdk_project_instance(experiment_key)
+      variation_key = optimizely_sdk_project_instance(experiment_key, "experiment")
       OptimizelyServerSide::Experiment.new(experiment_key, variation_key).start(&blk)
     end
     alias_method :feature_flip, :experiment
 
-    def optimizely_sdk_project_instance(experiment_key)
+    def optimizely_sdk_project_instance(experiment_key, type)
+      case type
+      when "experiment"
+        get_optimizely_instance.activate(experiment_key, visitor_id, user_attributes)
+      when "feature"
+        get_optimizely_instance.is_feature_enabled(experiment_key, visitor_id, user_attributes)
+      end  
+    end
+
+    def feature_test(experiment_key, options = {}, &blk)
```

Comment:
The method definition says it expects `blk` as well, but we are not making a `blk.call` or `yield` here. Is this accidental?

Reply:
Yeah, that was accidental, have removed